### PR TITLE
KevS-SK1392-Update-email-option-included-into-WebHosting

### PR DESCRIPTION
**Pull request by Kazarus4**
Updates : 

- date of "last update"
- add a warning paragraph about external domains, domain or DNS zone into another ovhcloud manager, domain already used by another webhosting for the email option

### CODE REVIEW

**Comment by Kazarus4** [comment]
need peer review for FRFR

File: pages/web_cloud/web_hosting/activate-email-hosting/guide.fr-fr.md [hunk 1]
```diff
@@ -18,6 +18,15 @@ Si vous avez commandé un hébergement web sans l'attacher directement à un nom
 - Posséder une [offre d'hébergement web](https://www.ovhcloud.com/fr/web-hosting/){.external}.
 - Être connecté à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr){.external}.
 
+> [!warning]
+>
+> Si vous souhaitez activer l'option « adresse(s) e-mail incluse(s) avec votre hébergement web » pour un nom de domaine externe (non enregistré chez OVHcloud), vous devrez, à minima, créer une zone DNS chez OVHcloud pour votre nom de domaine externe. Consultez notre guide « [Créer une zone DNS chez OVHcloud](/pages/web_cloud/domains/dns_zone_create) » si nécessaire. Ceci afin que votre nom de domaine apparaîsse dans la liste des noms de domaine disponibles lors de l'activation de l'option.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/activate-email-hosting/guide.fr-fr.md]:
```suggestion
> Si vous souhaitez activer l'option « adresse(s) e-mail incluse(s) avec votre hébergement web » pour un nom de domaine externe (non enregistré chez OVHcloud), vous devez créer (a minima) une zone DNS chez OVHcloud pour votre nom de domaine externe. Consultez notre guide « [Créer une zone DNS chez OVHcloud](/pages/web_cloud/domains/dns_zone_create) » si nécessaire. Cela permettra à votre nom de domaine d'apparaître dans la liste des noms de domaine disponibles lors de l'activation de l'option.
```

File: pages/web_cloud/web_hosting/activate-email-hosting/guide.fr-fr.md [hunk 2]
```diff
@@ -18,6 +18,15 @@ Si vous avez commandé un hébergement web sans l'attacher directement à un nom
...
+>
+> De plus, si votre nom de domaine est déjà enregistré ou si une zone DNS existe déjà pour ce dernier chez OVHcloud dans un autre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) que le vôtre, consultez notre documentation intitulée « [Gérer les contacts de ses services](/pages/account_and_service_management/account_information/managing_contacts) » pour en récupérer la gestion.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/activate-email-hosting/guide.fr-fr.md]:
```suggestion
> Par ailleurs, si votre nom de domaine est déjà enregistré chez OVHcloud ou si une zone DNS existe déjà pour ce dernier dans un autre [compte OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) que le vôtre, consultez notre documentation « [Gérer les contacts de ses services](/pages/account_and_service_management/account_information/managing_contacts) » pour en récupérer la gestion.
```

File: pages/web_cloud/web_hosting/activate-email-hosting/guide.fr-fr.md [hunk 3]
```diff
@@ -18,6 +18,15 @@ Si vous avez commandé un hébergement web sans l'attacher directement à un nom
...
+>
+> Enfin, si votre nom de domaine est déjà lié à l'option « adresse(s) e-mail incluse(s) avec votre hébergement web » d'un autre hébergement web, vous ne pourrez pas l'associer à votre propre hébergement web directement. Faites le point avec l'actuel gestionnaire de l'hébergement web accompagné du présent guide.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/activate-email-hosting/guide.fr-fr.md]:
```suggestion
> Enfin, si votre nom de domaine est déjà lié à l'option « adresse(s) e-mail incluse(s) avec votre hébergement web » d'un autre hébergement web, vous ne pourrez pas l'associer à votre propre hébergement web directement. Faites le point avec l'actuel gestionnaire de l'hébergement web en vous aidant de ce guide.
```

**Pull request merged**